### PR TITLE
Fix exit code check in Docker Compose tests

### PR DESCRIPTION
**Pull request by dominiklohmann**
These tests did not properly check for the exit code of all spawned containers since we started explicitly setting a container name. This fixes that oversight.

### CODE REVIEW

File: docker/test [hunk 1]
```diff
@@ -61,28 +61,28 @@ sleep 3
...
 
 # Print logs for easier debugging in case something went wrong.
-docker compose ${compose_files} ps
-docker compose ${compose_files} logs
+docker compose ps
+docker compose logs
 
 # Assert that all containers that exited have exited cleanly.
 exited_all="$(docker ps --all --filter "name=${COMPOSE_PROJECT_NAME}" --filter 'status=exited' -q | sort -u)"
 exited_cleanly="$(docker ps --all --filter "name=${COMPOSE_PROJECT_NAME}" --filter 'exited=0' -q | sort -u)"
```
Comment (rdettai) [inline docker/test:83]: Isn't `docker compose ps --all` an alternative to `docker ps --all --filter "name=${COMPOSE_PROJECT_NAME}"`. That would spare us the dance around the container name.

Comment (dominiklohmann) [reply docker/test:83]: TIL that `docker compose ps` exists. That does make a lot of things a lot easier, yes! I will refactor the script accordingly.

Comment (dominiklohmann) [reply docker/test]: I implemented this accordingly. This is so much better!

**Review by mavam — APPROVED** [review]
Looks like the feedback is addressed.

**Pull request merged**